_pages/home.md

```diff
@@ -57,11 +57,13 @@ peer-review:
          <div class="archive__item-body">
             <h2 class="archive__item-title">Peer review mentorship program</h2>
             <div class="archive__item-excerpt">
-               <p>We are building diverse community around the scientific packages that drive open science. We recruit and support editors and reviewers from different backgrounds and gender-identities in each review that we perform.
-            </p>
-              Are you new to peer review? We got you!
+               <p>
+               We are building diverse community around the scientific packages that drive open science. We recruit and support editors and reviewers from different backgrounds and gender-identities in each review that we perform.
+               </p>
+               <p>
```

Comment:
The font size issue that i thought was related to styles was simply a missing p tag.